pkg/cluster/admin_kind.go

```diff
@@ -131,6 +151,26 @@ func (a *kindAdmin) Create(ctx context.Context, desired *api.Cluster, registry *
 	return nil
 }
 
+func (a *kindAdmin) clusterExists(ctx context.Context, cluster string) (bool, error) {
+	buf := bytes.NewBuffer(nil)
+	cmd := exec.CommandContext(ctx, "kind", "get", "clusters")
```

Comment:
Is there any sort of format (e.g. `-ojson`) arg we could use here?

Reply:
surprisingly not!!
https://github.com/kubernetes-sigs/kind/blob/main/pkg/cmd/kind/get/clusters/clusters.go#L34